_posts/alm7468/2022-09-28-Focused-Reflection-alm7468.md

```diff
@@ -1,3 +1,8 @@
+---
+layout: post
+author: alm7468
+title: April's Focused Reflection
```

Comment:
Put quotes " " around the title please. 
Should read "April's Focused Reflection"